code-challenges.test.js

```diff
@@ -46,7 +79,14 @@ var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 var randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
-
+//Declare a function named capWords with a parameter named array.
+const capWords = (array) => {
+//create a variable for the new mapped array
+    let newArr = array.map(value => {
+//access the first index and capitalize the first letter for each word.
+        return value[0].toUpperCase()
```

Comment:
Between lines 87 and 114 I can see you have the right idea. The split() method has an opposite called the join() method. it works in similar ways. Definitely worth the research.